js/util.js

```diff
@@ -134,19 +134,22 @@
 
     // angle between two vectors in radians
     function angle(v1, v2){
-        var diff = v1.rad - v2.rad;
+        var diff = v1.radians() - v2.radians();
         return atan2(sin(diff), cos(diff));
     }
 
     // Create a vector from an angle in degrees and a magnitude (length)
     function Vector(x,y){
-        this.x = x;
-        this.y = y;
+        this.x = Number(x);
+        this.y = Number(y);
```

Comment:
Yes. We should (and do, when it works) handle type conversions elsewhere. This was added when I was getting errors, but the error was from something different and has been fixed. On the one hand, a little defensive type conversion might not be a bad thing, on the other it has a small performance impact and a larger inconsistent style impact. I'll back it out.